pkg/scheduler/controller/shoot/reconciler.go

```diff
@@ -21,7 +21,7 @@ import (
 
 	gardencorev1beta1 "github.com/gardener/gardener/pkg/apis/core/v1beta1"
 	gardencorev1beta1helper "github.com/gardener/gardener/pkg/apis/core/v1beta1/helper"
-	gardenversionedcoreclientset "github.com/gardener/gardener/pkg/client/core/clientset/versioned"
+	externalcoreclientset "github.com/gardener/gardener/pkg/client/core/clientset/versioned"
```

Comment:
Why is this fixing failing unit tests? 👀 I only see the import alias being changed, how does this have any effect?